src/api/java/blusunrize/immersiveengineering/api/wires/utils/WireUtils.java

```diff
@@ -251,4 +233,21 @@ public static void moveConnectionEnd(Connection conn, ConnectionPoint currEnd, C
 		globalNet.removeConnection(conn);
 		globalNet.addConnection(new Connection(conn.type, fixedPos, newEnd));
 	}
+
+	public interface RaytraceCallback
+	{
+		HashSet<BlockPos> rayTrace(Vector3d start, Vector3d end, World world, Consumer<BlockPos> out);
+	}
+
+	//TODO dedup with Utils#…
```

Comment:
Are you going to resolve this TODO as part of this PR, or in a later step?

Reply:
I'll check it later today, that's definitely for this PR. I'm pretty sure I've resolved the TODO at some point, either I forgot to remove it after resolving it or that change got lost somewhere.